klog_file.go

```diff
@@ -124,3 +126,14 @@ func create(tag string, t time.Time) (f *os.File, filename string, err error) {
 	}
 	return nil, "", fmt.Errorf("log: cannot create log: %v", lastErr)
 }
+
+// The startup argument indicates whether this is the initial startup of klog.
+// If startup is true, existing files are opened for apending instead of truncated.
+func openOrCreate(name string, startup bool) (*os.File, error) {
+	if startup {
+		f, err := os.OpenFile(name, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
```

Comment:
is this the same mode that os.Create uses?

Reply:
are the same flags correct cross-platform?

Reply:
Yes, both use `0666`. https://golang.org/src/os/file.go#L274

Go generally tries its best. On Windows there's some magic to translate the ones used here into the appropriate parameters: https://golang.org/src/syscall/syscall_windows.go#L248